server/server.go

```diff
@@ -576,6 +606,29 @@ func (r *rodeServer) UpdatePolicy(ctx context.Context, updatePolicyRequest *pb.U
 	return policy, nil
 }
 
+// validateRodeRequirementsForPolicy ensures that these two rules are followed:
+// 1. A policy is expected to return a pass that is simply a boolean representing the AND of all rules.
+// 2. A policy is expected to return an array of violations that are maps containing a description id message name pass. pass here is what will be used to determine the overall pass.
+func (r *rodeServer) validateRodeRequirementsForPolicy(ctx context.Context, regoContent string) []error {
```

Comment:
Does this function need `ctx` or the `rodeServer`? It doesn't look like they're used. 

Would this benefit from parsing as well? It looks like there's a way to get the variables in the policy:

```go

for _, r := range a.Rules {
	fmt.Println(r.Head.Name)
}
```
I didn't see any type information though.

Reply:
Yeah I think I like parsing better than relying on regex both here and in the other comment.